### PR TITLE
[Release] Fix release tests

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?

We should use bash run.sh to run release tests now. This PR fixes all broken release tests. 

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

**Comment by simon-mo** [comment]
What's the reasoning? Do we no longer fill those in with ray project template?

**Comment by rkooo567** [comment]
Yeah Run commands in project yaml has been deprecated, and we cannot use it anymore.

**Review by simon-mo — APPROVED** [review]

File: ci/long_running_distributed_tests/ray-project/project.yaml [hunk 1]
```diff
@@ -5,9 +5,14 @@ name: long-running-distributed-tests
...
-      help: "URL to the ray wheel to test (defaults to latest)"
-      default: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.9.0.dev0-cp36-cp36m-manylinux1_x86_64.whl
+    - name: ray_version # Ray version string.
+      default: "0.8.2"
+
+    - name: commit # Ray commit SHA string.
+      default: "f5a1307a608fe5fdbdb04616b22c91f029af329a"
+
+    - name: ray_branch
+      default: "releases/0.8.2"
```
Comment (simon-mo) [inline ci/long_running_distributed_tests/ray-project/project.yaml:15]: should these be deleted then?

Comment (rkooo567) [reply ci/long_running_distributed_tests/ray-project/project.yaml:15]: I'd like to just keep it until APIs for CLI becomes stabilized.

**Comment by rkooo567** [comment]
@simon-mo Can you merge it? the test result shouldn't be related.

**Pull request merged**